src/Microsoft.AspNet.Mvc.Core/ModelBinding/PrefixContainer.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using Microsoft.Framework.Internal;
 
 namespace Microsoft.AspNet.Mvc.ModelBinding.Internal
```

Comment:
Making this public and not in the internal namespace. You'd need this to implement a value provider correctly.